test/unit/specs/components/staking/__snapshots__/PageBond.spec.js.snap

```diff
@@ -176,7 +176,7 @@ exports[`PageBond disables bonding if not connected 1`] = `
                         class="bond-value__input tm-field"
                         disabled="disabled"
                         id="new-unbonded-atoms"
-                        placeholder="Steaks"
+                        placeholder="0"
```

Comment:
placeholder="0" ?! /1 ? 1?!?!?!

Reply:
i think having `Steaks` or `Atoms` would be misleading the user to think they should enter a denomination when really the input is expecting a number.